interop/src/main.rs

```diff
@@ -165,9 +169,10 @@ async fn main() {
             port: opt.port.unwrap_or(4433),
             retry_port: opt.retry_port.unwrap_or(4434),
             sequential: opt.seq,
-            alpn: match (opt.h3, opt.hq) {
-                (false, true) => Alpn::Hq,
-                (true, false) => Alpn::H3,
+            alpn: match (opt.h3, opt.hq, opt.siduck) {
+                (false, true, _) => Alpn::Hq,
+                (true, false, _) => Alpn::H3,
+                (_, _, true) => Alpn::Siduck,
```

Comment:
I think we should get rid of the `_`s here, except for a fallthrough case that emits an error (or just panics).